.coverage

```diff
@@ -1 +1 @@
-!coverage.py: This is a private format, don't read it directly!{"lines":{"/Users/georgesmith-sweeper/PythonTTT/test_user.py":[1,2,4,8,13,18,23,5,6,9,10,11,14,15,16,19,20,21,24,25,26,27,28,29,30],"/Users/georgesmith-sweeper/PythonTTT/board.py":[1,2,4,9,15,21,26,32,5,6,7,16,19,22,23,24],"/Users/georgesmith-sweeper/PythonTTT/user.py":[1,2,4,5,8,11,14,30,9,15,16,17,19,23,24,28,20,21,31,32,33],"/Users/georgesmith-sweeper/PythonTTT/ui.py":[1,2,3]}}
\ No newline at end of file
+!coverage.py: This is a private format, don't read it directly!{"lines":{"/Users/georgesmith-sweeper/PythonTTT/test_board.py":[1,2,3,5,9,18,30,42,48,54,60,68,77,86,94,101,105,109,123,134,145,6,7,10,11,12,19,20,21,31,32,33,34,35,36,43,44,45,46,49,50,51,52,55,56,57,58,61,62,63,64,65,66,69,71,72,73,75,78,80,81,82,84,87,88,89,90,91,95,96,97,98,99,102,103,106,107,110,111,112,124,125,126,128,129,130,132,135,136,137,139,140,141,143,146,147,148,150,151,152,154],"/Users/georgesmith-sweeper/PythonTTT/board.py":[1,2,5,10,11,15,25,34,40,50,65,66,69,79,80,83,86,87,3,6,7,8,88,89,90,81,84,67,12,13,16,17,18,19,20,21,22,23,26,27,28,29,30,31,32,35,36,37,38,41,42,43,44,45,46,47,48,51,52,53,54,56,57,59,60,62,63,70,71,72,73,74,75,76],"/Users/georgesmith-sweeper/PythonTTT/test_start.py":[1,2,3,5,16,6,8,9,10,12,13,14,17,19,20,21,23,24,25],"/Users/georgesmith-sweeper/PythonTTT/user.py":[1,2,4,5,7,10,15,16,8,17,18,19,21,25,26,30,22,23,11,12,13],"/Users/georgesmith-sweeper/PythonTTT/ui.py":[1,3,4,7,8,11,12,13,14,15,16,17,18,20,21,24,25,26,27,28,29,30,31,32,33,5],"/Users/georgesmith-sweeper/PythonTTT/start.py":[1,2,3,4,6,33,7,9,10,11],"/Users/georgesmith-sweeper/PythonTTT/test_ui.py":[1,2,3,4,7,22,8],"/Users/georgesmith-sweeper/PythonTTT/test_user.py":[1,2,4,8,13,18,23,5,6,9,10,11,14,15,16,19,20,21,24,25,26,27,28,29,30]}}
```

Comment:
You probably don't want to publish this to version control. same goes for all your `.pyc` files below. Look into adding a `.gitignore` to your project and removing these files from your git history. I'd also like for you to understand why these types of files are typically omitted from version control.